src/site-server.ts

```diff
@@ -20,7 +20,7 @@ import {
 import WpCliProcess, { MessageCanceled, WpCliResult } from 'src/lib/wp-cli-process';
 import { createScreenshotWindow } from 'src/screenshot-window';
 import { getSiteThumbnailPath } from 'src/storage/paths';
-import { Blueprint } from './stores/wpcom-api';
+import { Blueprint } from 'src/stores/wpcom-api';
```

Comment:
Nit: This helps a bit with readibility
```suggestion
import type { Blueprint } from 'src/stores/wpcom-api';
```